sql.txt

```diff
@@ -1,14 +1,40 @@
 (1) Using the Customer Table, select the CustomerName, ContactName, and Country.  Use ORDER BY to order by Country.  
 Use LIMIT and OFFSET to get entries 11 through 20. Paste your SQL statement below. 
+SELECT CustomerName, ContactName, Country 
+FROM Customers 
+ORDER BY Country 
+LIMIT 10 
+OFFSET 10;
 
 (2) Select all columns from the Customer table where the ContactName starts with A.  Paste your SQL statement below.
+SELECT * 
+FROM Customers
+WHERE ContactName LIKE 'A%';
 
 (3) Select all columns from the OrderDetails table where the ProductID is 51 and the quantity is greater than 10.
 Paste your SQL statement below.
 
+SELECT * 
+FROM OrderDetails 
+WHERE ProductID=51
+AND Quantity > 10;
+
 (4) Insert 3 rows into the Products table.  Note that you will have to specify a valid SupplierID and CategoryID, 
 corresponding to rows from the Supplier and Category tables.  Paste your three SQL statements below.
+INSERT INTO Products(ProductName, SupplierID, CategoryID, Unit, Price)
+    VALUES ('Product1', 1, 1, "500 ml", 10),
+           ('Product2', 2, 2, "5 kg pkg.", 10),
+           ('Product3', 3, 3, "12 boxes", 20);
 
```

Comment:
This looks great. You seem to understand SQL very well. 

Just one small recommendation. I would stick to either single or double quotes within the query - doesn't matter which one, as that will change when coding in different languages. However, it is important to stay consistent within code and avoid any confusion, so either

`VALUES ('Product1', 1, 1, '500 ml', 10)`
or
`VALUES ("Product1", 1, 1, "500 ml", 10)`

Hope that makes sense!